test/StdChains.t.sol

```diff
@@ -27,6 +27,7 @@ contract StdChainsTest is Test {
         assertEq(getChain(1).rpcUrl, "https://mainnet.infura.io/v3/b1d3925804e74152b316ca7da97060d3");
         assertEq(getChain("optimism_goerli").rpcUrl, "https://goerli.optimism.io/");
         assertEq(getChain("arbitrum_one_goerli").rpcUrl, "https://goerli-rollup.arbitrum.io/rpc/");
+        assertEq(getChain("metis").rpcUrl, "https://andromeda.metis.io/?owner=1088");
```

Comment:
This is in the wrong section, you can see from the comment this section is testing RPC URLs that have overrides in `foundry.toml`. If you just want to test that the metic RPC is correct, we should move this line to the `// Other RPCs should remain unchanged` portion of this test